apps/central/src/components/Groups.vue

```diff
@@ -282,6 +299,13 @@ export default {
       }
     },
   },
+  watch: {
+    showChangeColumn: function (val) {
```

Comment:
usually we use the ()=>{} style function in the watches, why this time function(){} ?